applications/xe-cnf/helm/n3iwf-n2fe/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
This file doesn't have Copyright header. This is helm configuration file. There is no need to have this file in repo. Can you please remove this file?